gapis/gfxapi/gfxapi.proto

```diff
@@ -50,6 +50,9 @@ enum ResourceType {
 	ShaderResource = 5;
 	// ProgramResource represents the Program resource type
 	ProgramResource = 6;
+	// UninitializedTextureResource represents texture resource type
+	// which has been generated but does not have dimensions yet.
+	UninitializedTextureResource = 7;
```

Comment:
I am also aware of that, but in that case the resource RPC is wrong in the first place...